src/components/TimePicker/TimePicker.tsx

```diff
@@ -73,13 +82,13 @@ function insertAtPosition(originalString: string, newSubstring: string, from: nu
  */
 function replaceRangeWithZeros(originalString: string, from: number, to: number): string {
     const normalizedFrom = Math.max(from, 0);
-    const normalizedTo = Math.min(to, 2);
+    const normalizedTo = Math.min(to, 3);
```

Comment:
Are we sure this change does not have any sideeffects in other parts of the app?